static/_redirects

```diff
@@ -633,6 +633,10 @@ support/how-to/rackspace-email-password-recovery-faq/  /support/how-to/manage-pa
 
 # Individual Redirects
 
+
+/support/how-to/cloud-servers-attaching-and-detaching-networks/  /how-to/attach-and-detach-networks-from-a-cloud-server/  404
+/how-to/attach-and-detach-networks-from-a-cloud-server/  /support//how-to/attach-and-detach-networks-from-a-cloud-server/  301  
```

Comment:
You've got an extra forward slash in the last bit there "/support//how-to/..."

Reply:
ohhh haha thanks